engine/class_modules/sc_druid.cpp

```diff
@@ -10606,6 +10607,14 @@ void druid_t::shapeshift( form_e f )
   form = f;
 }
 
+// druid_t::find_target_data ================================================
+
+const druid_td_t* druid_t::find_target_data( const player_t* target ) const
+{
+  assert( target );
```

Comment:
Is there a specific reason that the druid override has an assert in it, and other do not?
I mean if we really care we should maybe at some point start annotation pointers we think should not be null. (gsl not_null maybe)

Reply:
Wanted to keep it in line with the get_target_data stuff druids are doing. But no, it's not strictly necessary.